packages/block-editor/src/components/publish-date-time-picker/style.scss

```
@@ -0,0 +1,20 @@
+.block-editor-publish-date-time-picker__header {
+	margin-bottom: 1em;
+}
+
+.block-editor-publish-date-time-picker__heading {
+	font-size: $default-font-size;
+	margin: 0;
+}
+
+.block-editor-publish-date-time-picker__reset {
+	height: $icon-size;
+	margin: 0;
+	text-decoration: underline;
+}
+
+.block-editor-publish-date-time-picker__close.has-icon {
```

I wonder if there's a way to apply these styles that doesn't involve using the `has-icon` classname, which should be considered as an implementation detail of the `Button` component

---

Do you think the `&&&` hack would be better? It's just specificity that we need.

---

I think so — the `&&&` hack is more upfront about why is being used (specificity) and would allow us to avoid using hardcoded internal components classnames :)

---

Hm, doesn't look like `sass` lets you do the `&&&` hack. I think how it is currently is the best we can do.

```
Error: "&" may only used at the beginning of a compound selector.
```

---

Off the top of my head, I remember being able to do something like that by using [interpolation](https://sass-lang.com/documentation/interpolation) — something like

```sass
  #{ & }#{ & }#{ & } {
    // styles
  }
```

although I'm not 100% sure

---

The hack I usually reach for is attribute selectors.
```suggestion
[class].block-editor-publish-date-time-picker__close {
```
You can repeat the same one as required for specificity but here you might only need the single.